e2e/crm7/pages/provider/claim-details.js

```diff
@@ -13,6 +13,10 @@ export default class ClaimDetailsPage {
         await this.page.getByRole('group', { name: 'Did you spend time watching' }).getByLabel('No').check();
         await this.page.getByRole('group', { name: 'Did you do any work before' }).getByLabel('No').check();
         await this.page.getByRole('group', { name: 'Did you do any further work' }).getByLabel('No').check();
+        // Date work completed
```

Comment:
Just curious but Is there an expectation to leave comments for this sort of change?